pages/social-inbox/activity-pub.mdx

```diff
@@ -0,0 +1,170 @@
+# Activity Pub
+
+The Social Inbox implements the [ActivityPub
+specification](https://www.w3.org/TR/activitypub/)'s dynamic parts,
+such as handling incoming notifications, follow requests, likes, boosts,
+and sending out notifications, as well as related specifications, like
+[Webfinger](https://webfinger.net/)
+
+## Inbox
+
+The [Inbox](https://www.w3.org/TR/activitypub/#inbox) receives
+activities from other actors and remote instances, such as Like,
+Announce, Follow, and replies in the form of Create and Update.
+
+Depending on your blocklist and allowlist, activities may be put on
+pause, accepted or rejected.
+
+The Inbox is a paginated collection that holds any activity received and
+stored, irrespective of their paused/approved status.  Only query the
+Inbox to check what activities are there.
+
+Rejected activities are currently not stored anywhere.
+
+Example: <https://social.distributed.press/v1/@distributed@distributed.press/inbox>
+
+## Outbox (notifications)
+
+To notify followers of your activities, send them as a POST request to
+your outbox.
+
+## Followers
+
+The [Followers](https://www.w3.org/TR/activitypub/#followers) collection
+contains a list of remote Actors that have sent a Follow request and was
+approved.
+
+When you send a Create/Update/Delete activity to your outbox, followers
+are notified on their inboxes.
+
+## Interacted
+
+This is an out-of-spec and internal collection that keeps track of any
+actor that has ever interacted with your activities, even if they don't
+follow you anymore.
+
+It helps the Social Inbox notify remote instances when an activity is
+deleted.
+
+## Handled activities
+
+Activities the Social Inbox processes.
+
+### Follow
+
+When a Follow request is received, and is approved (either automatically
+or manually), the Actor is added to the followers collection and the
+Social Inbox sends out the activities from the outbox to backfill the
+profile.
+
+When the Follow request is rejected, the remote instance receives a
+Reject activity.
+
+### Create/Update
+
+Create/Update activities are added to the Inbox when their actor is not
+blocklisted.
+
+When rejected, the Social Inbox will receive the activity, notify the
+onRejected webhook, remove it from the Inbox, but otherwise won't notify
+the remote instance with a Reject activity.
+
+### Delete
+
+When a Delete activity is received, it gets auto-approved regardless of
```

Comment:
Delete (and undo) activities are only accepted for objects authored by the actor performing the delete